frontend/src/router.js

```diff
@@ -0,0 +1,214 @@
+                useLayout: true,
+                load: () => {
+                    new IncomeOutcomeForm('create');
+                }
+            },
+            {
+                route: '/edit-income-outcome',
+                title: 'Редактировать доход/расход',
+                template: '/templates/edit-income-outcome.html',
+                useLayout: true,
+                load: () => {
+                    new IncomeOutcomeForm('edit');
+                }
+            },
+            {
+                route: '/income',
+                title: 'Доходы',
+                template: '/templates/income.html',
+                useLayout: true,
+                load: () => {
+                    new Income();
+                }
+            },
+            {
+                route: '/expenses',
+                title: 'Расходы',
+                template: '/templates/expenses.html',
+                useLayout: true,
+                load: () => {
+                    new Expenses();
+                }
+            },
+            {
+                route: '/income-expense-table',
+                title: 'Таблица доходов и расходов',
+                template: '/templates/income-expense-table.html',
+                useLayout: true,
+                load: () => {
+                    new IncomeExpensesTable();
+                }
+            }
+        ];
+    }
+
+    initEvents() {
+        window.addEventListener('DOMContentLoaded', this.activateRoute.bind(this));
+        window.addEventListener('popstate', this.activateRoute.bind(this));
+    }
+
+    loadGlobalStyles() {
+        const head = document.head;
+        const addResource = (type, hrefOrSrc) => {
+            const tag = document.createElement(type);
+            if (type === 'link') {
+                tag.rel = 'stylesheet';
+                tag.href = hrefOrSrc;
+            } else if (type === 'script') {
+                tag.src = hrefOrSrc;
+                tag.defer = true;
+            }
+            head.appendChild(tag);
+        };
+
+        // Подключаем Bootstrap
+        addResource('link', '/libs/bootstrap/bootstrap.css');
+        addResource('script', '/libs/bootstrap/bootstrap.bundle.js');
+
+        // Подключаем общие стили
+        addResource('link', '/styles/common.css');
+        addResource('link', '/styles/adaptive.css');
+
+    }
+
+    async activateRoute() {
+        const urlRoute = window.location.pathname;
+        const newRoute = this.routes.find(item => item.route === urlRoute);
+
+        if (newRoute) {
+            if (newRoute.title) {
+                this.titlePageElement.innerText = newRoute.title + ' | Lumincoin Finance';
+            }
+
+            if (newRoute.template) {
+                if (newRoute.useLayout) {
+                    this.loadLayoutStyles();
+                    const layoutHTML = await fetch('/templates/layout.html').then(response => response.text());
```

Comment:
```suggestion
                    this.contentPageElement.innerHTML = await fetch('/templates/layout.html').then(response => response.text());
```
а строчку ниже удали

Reply:
Сделано